src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -271,11 +271,11 @@
     updateNamingExclusions: function(component, hh) {
         var strExclusions = '';
         if (!component.get('v.isAutoName'))
-            strExclusions += 'Household__c.Name;';
```

Comment:
turns out the picklist values don't include Household__c.  the just had the field names.  The managehouseholdui code (from ryan) was setting household__c as well, but it didn't matter, because the code that looks at this field, was just doing a contains(fieldname).

So I fixed up the new managehouseholdui code to set the correct picklist values.